haas/test_common.py

```diff
@@ -127,6 +127,7 @@ def newDB():
 def releaseDB():
     """Do we need to do anything here to release resources?"""
     with app.app_context():
+        db.reflect()
```

Comment:
So, that kind of error can happen if the database and the running code disagree on what the schema (equivalently, set of models) should be. This very well might happen if the migration tests are failing; that's half of what they're there to test. I wouldn't be surprised if, having fixed those tests, this was no longer necessary. Furthermore, if this *is* failing it's probably an actual bug of some sort. Can you try taking it out and see if it still fails?